src/Components/Search/Previews/Grids/ArtistSearch/MarketingCollections.tsx

```diff
@@ -0,0 +1,101 @@
+import { data as sd } from "sharify"
+
+import { Box, color, Flex, Link, Sans, Serif, space } from "@artsy/palette"
+import { MarketingCollectionsPreview_marketingCollections } from "__generated__/MarketingCollectionsPreview_marketingCollections.graphql"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import styled from "styled-components"
+
+interface MarketingCollectionsPreviewProps {
+  marketingCollections: MarketingCollectionsPreview_marketingCollections
+}
+
+const CollectionBox = styled(Box)<{ imageUrl: string }>`
+  margin-right: ${space(2)}px;
+  margin-bottom: ${space(2)}px;
+  width: 185px;
+  height: 80px;
+  background: linear-gradient(to bottom, rgba(0, 0, 0, 0.4), rgba(0, 0, 0, 0.4)),
+    url(${p => p.imageUrl}) center;
+
+  &:hover {
+    background: linear-gradient(
+        to bottom,
+        rgba(0, 0, 0, 0.5),
+        rgba(0, 0, 0, 0.5)
+      ),
+      url(${p => p.imageUrl}) center;
+  }
+`
+
+const CollectionTitle = styled(Serif).attrs({ display: "inline" })`
+  color: ${color("white100")};
+`
+
+export const CollectionTitles = ({ title }: { title: string }) => {
+  const [first, ...rest] = title.split(": ")
+  const showHeadline = rest.length !== 0
+  const headline = showHeadline && first
+  const subtitle = showHeadline ? rest.join(": ") : first
+
+  return (
+    <Flex justifyContent="flex-end" flexDirection="column" height="80px">
+      {showHeadline && (
+        <Box ml={`${space(1)}px`} mb={`-${space(0.5)}px`}>
```

Comment:
```
<Box ml={1} mb={.5}>
```
These props  are pulled in automatically via our [Theme](https://github.com/damassi/gatsby-starter-docs/blob/master/src/Theme.tsx) and don't need to be translated with helpers.

Reply:
Yeah, I realized way late in our work that we were making these harder than they need to be. I'll circle back and update them!

Reply:
Strangely @damassi....wherever I change these to use an integer instead of an interpolated string, I get an unexpected increment. 

For instance - ``ml={`${space(1)}px`}`` renders as a 10px margin...but converting it to `ml={1}`, I get a 4px margin instead of 10. `ml={2}` gives me 8px; `ml={3}` gives me 16px. It's happening in all of these components.

Everything I've seen convinces me that I should expect 1->10px, 2->20px, and 3->30px. I blew away my node_modules and re-yarned, to make sure it wasn't something with an older version of styled-components, and it still happens.

1. Are my expectations correct? 
2. If this is unexpected behavior, is there anything you can think of that would cause this to happen? Is there any sort of "activation" of the theme that needs to happen, to pull in the correct increments, that we might be missing?

Reply:
It does resolve if I wrap a `<Theme>` component around my story. I guess I'm surprised that no other stories do that.

Reply:
Oh interesting... I wonder if you're using the right `storiesOf` helper... 👁

Reply:
Nope 😀

Good call, and thank you!